mmv1/products/securesourcemanager/Instance.yaml

```diff
@@ -80,6 +80,13 @@ examples:
       'deletion_policy': '"DELETE"'
     ignore_read_extra:
       - 'update_time'
+  - name: 'secure_source_manager_instance_private_trusted_cert'
```

Comment:
There are already a lot of examples here, and this one doesn't seem to add much value. Would it be possible to update a different example to not set this field? Otherwise, you could set `exclude_docs` to only generate a test.